pootle/apps/pootle_misc/templatetags/common_tags.py

```diff
@@ -36,14 +37,20 @@ def avatar(username, email_hash, size):
 @register.inclusion_tag('browser/_progressbar.html')
 def progress_bar(total, fuzzy, translated):
     if not total:
-        return dict(fuzzy=0, translated=0, untranslated=0)
-    untranslated = total - translated - fuzzy
+        fuzzy = translated = untranslated = 0
+    else:
+        untranslated = total - translated - fuzzy
+        untranslated = float(untranslated)/total
+        fuzzy = float(fuzzy)/total
+        translated = float(translated)/total
+    cldrformat = "#,##0.0%"
     return dict(
-        fuzzy=round((float(fuzzy)/total) * 100),
-        translated=round(
-            (float(translated)/total) * 100),
-        untranslated=round(
-            (float(untranslated)/total) * 100))
+        untranslated=cldrpercent(untranslated, cldrformat),
+        fuzzy=cldrpercent(fuzzy, cldrformat),
+        translated=cldrpercent(translated, cldrformat),
+        bar_u=round(untranslated * 100),
+        bar_f=round(fuzzy * 100),
+        bar_t=round(translated * 100))
```

Comment:
I look to rename, but its was getting noisy.

I'm not sure what "nothing is returned here means"

CLDR functions need a fraction ie 0.956 to create 96,6%.  While the bar needs a perecent expressed as 95.6%